release-notes/9.0/preview/preview5/api-diff/Microsoft.NETCore.App/9.0-preview5_System.Diagnostics.Metrics.md

```diff
@@ -0,0 +1,9 @@
+# System.Diagnostics.Metrics
+
+``` diff
+ namespace System.Diagnostics.Metrics {
+-    public delegate void MeasurementCallback<T>(Instrument instrument, T measurement, ReadOnlySpan<KeyValuePair<string, object?>> tags, object? state);
++    public delegate void MeasurementCallback<T>(Instrument instrument, T measurement, ReadOnlySpan<KeyValuePair<string, object?>> tags, object? state) where T : struct;
+ }
```

Comment:
There is no real change here. this didn't really changed in this release.

Reply:
The diff says this changed:

> `where T : struct`

Do we not care about that? Or are you saying this was already there before?

Reply:
> Or are you saying this was already there before?

yes, this was there before.

Reply:
From 8.0 release branch: 

https://github.com/dotnet/runtime/blob/1537de64fce3a42991e5161109f1c1245cb8865e/src/libraries/System.Diagnostics.DiagnosticSource/src/System/Diagnostics/Metrics/MeterListener.cs#L12